recipes/nibabel/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install     # Python command to install the script.
```

Comment:
As it appears we could be using `setuptools`, could you please add these arguments?

     --single-version-externally-managed --record record.txt